### PR TITLE
Allow to navigate by multiple years / issue #102

### PR DESCRIPTION
**Pull request by MatthieuBarthel**
Here is a first shot :

1. Ok, the step is configurable.
2. Ok, the feature is disabled by default.
3. Ok, the buttons behavior with min/max dates seems fine to me.
4. Not ok, I tried to add some e2e tests but I cannot trigger a click event on the fast/forward buttons, I get the message `Failed: element not visible`. I can't figure out what's going on, I see the buttons on the screen while running the tests but what ever I tried it fails. Could you help ?

Waiting for your feedback, don't hesitate if you want me to do some update, like renaming variables :-)

Thank you !

### CODE REVIEW

File: src/app/calendar-nav/calendar-nav.component.less [hunk 1]
```diff
@@ -16,11 +16,25 @@
...
+
+  .dp-calendar-nav-container-left {
+    left: 5px;
+  }
+
+  .dp-calendar-nav-container-right {
+    right: 5px;
+  }
+
+  .dp-calendar-nav-left, .dp-calendar-nav-right, .dp-calendar-secondary-nav-left, .dp-calendar-secondary-nav-right {
```
Comment (vlio20) [inline src/app/calendar-nav/calendar-nav.component.less]: please break to lines

File: src/app/calendar-nav/calendar-nav.component.less [hunk 2]
```diff
@@ -30,11 +44,15 @@
 
   .dp-calendar-nav-left {
     .arrow(-135deg);
-    left: 5px;
   }
 
-  .dp-calendar-nav-right {
-    right: 5px;
+  .dp-calendar-secondary-nav-left::before, .dp-calendar-secondary-nav-right::before {
+    content: "«";
```
Comment (vlio20) [inline src/app/calendar-nav/calendar-nav.component.less]: Icons should be made with pure CSS in order to support different OS

File: src/app/calendar-nav/calendar-nav.component.less [hunk 3]
```diff
@@ -30,11 +44,15 @@
...
+    font-size: 16px;
+  }
+
+  .dp-calendar-secondary-nav-right::before {
+    content: "»";
```
Comment (vlio20) [inline src/app/calendar-nav/calendar-nav.component.less]: same here

File: src/app/calendar-nav/calendar-nav.component.less [hunk 4]
```diff
@@ -55,7 +73,7 @@
       border: 1px solid @c-light-gray;
     }
 
-    .dp-calendar-nav-left, .dp-calendar-nav-right {
+    .dp-calendar-nav-left, .dp-calendar-nav-right, .dp-calendar-secondary-nav-left, .dp-calendar-secondary-nav-right {
```
Comment (vlio20) [inline src/app/calendar-nav/calendar-nav.component.less]: break to lines pls

File: src/app/month-calendar/month-calendar.component.ts [hunk 5]
```diff
@@ -163,19 +163,45 @@ export class MonthCalendarComponent implements OnInit, OnChanges, ControlValueAc
...
+  }
+
   onRightNav() {
     this.currentDateView.add(1, 'year');
     this.yearMonths = this.monthCalendarService.generateYear(this.currentDateView, this.selected);
   }
 
+  onRightSecondaryNav() {
+    let navigateBy = this.componentConfig.multipleYearsNavigateBy
+    if (this.componentConfig.max && this.componentConfig.max.year() - this.currentDateView.year() < navigateBy) {
```
Comment (vlio20) [inline src/app/month-calendar/month-calendar.component.ts]: lets put this if logic into a variable

**Comment by vlio20** [comment]
@MatthieuBarthel, are you planning to fix the comments?

**Comment by MatthieuBarthel** [comment]
Thank you for your feedback and sorry for the late answer. I just pushed an update :
1. I first pushed with the x-prefix, tests went fine. As I had many tests failing locally, I tried to remove the prefix, but it failed: https://travis-ci.org/vlio20/angular-datepicker/builds/259683865 
2. What is missing ? I updated the README and demo.
3. Fixed.

Let me know :-)

File: src/app/calendar-nav/calendar-nav.component.html [hunk 6]
```diff
@@ -1,21 +1,37 @@
...
-  <button type="button"
-          class="dp-calendar-nav-right"
-          [hidden]="!showRightNav"
-          [disabled]="rightNavDisabled"
-          (click)="rightNavClicked()">
-  </button>
+  <div class="dp-calendar-nav-container-right">
+    <button type="button"
+            class="dp-calendar-secondary-nav-right"
+            [hidden]="!showRightSecondaryNav"
```
Comment (vlio20) [inline src/app/calendar-nav/calendar-nav.component.html]: why not to use *ngIf

File: src/app/common/styles/variables.less [hunk 7]
```diff
@@ -20,3 +20,21 @@
     transform: rotate(@deg);
   }
 }
+
+.double-arrow(@deg) {
```
Comment (vlio20) [inline src/app/common/styles/variables.less]: can you please post an image of this?

**Comment by MatthieuBarthel** [comment]
Here are the double arrows :

image

**Pull request merged**